data/assets/scene/solarsystem/missions/voyager/voyager1.asset

```diff
@@ -195,8 +197,10 @@ local VoyagerTrailCruiseSaturnInf = {
     Color = { 0.70, 0.50, 0.20 },
     StartTime = "1980 NOV 16",
     EndTime = "2100 JAN 01",
-    -- 43511 is the number of days between the Start and End time
-    SampleInterval = 43511 * 2
+    -- 1 sample every 2d
+    SampleInterval = openspace.duration("1980 NOV 16", "2100 JAN 01") /
```

Comment:
This calculation seems to be very common, in fact, every time the `duration` function is used in this PR it is used for this calculation. Isn't it better then to make a Lua function that does all of these steps instead of just one of them? It decreases the number of lines of code and will be easier to maintain.

Reply:
Maybe we could have the function be named more to what it is used for, such as `getSampleInterval(...)` or `calculateSampleInterval(...)`. Since the function will almost solely be used for this purpose I don't see a problem with naming it for that specifically.